### ISSUE TITLE
I think i found the sniping bug!

### ISSUE THREAD

**Issue by temel38** [NECROBOTIO/NecroBot#4122]
When i set this config, the bot doenst snipe pokemon and it just loops over and over; 
"GetSniperInfoFromPokezz": true,
"GetOnlyVerifiedSniperInfoFromPokezz": false,
"GetSniperInfoFromPokeSnipers": **true**, 

To fix it i set the config like this;
"GetSniperInfoFromPokezz": true,
 "GetOnlyVerifiedSniperInfoFromPokezz": false,
 "GetSniperInfoFromPokeSnipers": **false**,

So i think there is a bug when necrobot gets coords from both "pokezz" and "pokesnipers"
I hope I helped you guys by this..

**Comment by Marsinator** [comment]
#4003

potential fix/duplicate

**Comment by jjskuld** [comment]
Once #4111 is merged, then you can snipe from both PokeZZ and Pokesnipers to get more pokemon.  Duplicates will just be skipped over quickly.

**Comment by jjskuld** [comment]
Yes, the functionality overlaps with pogofeeder.  Now, if you just want to snipe from PogoZZ or PokeSnipers, you can do so without the extra dependency on pogofeeder.
